test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -155,14 +35,53 @@ def load_schemas_to_dict():
         new_schema_dict[s.name].append(s)
     return new_schema_dict
 
+def has_valid_upgraders(schema, version_map):
```

Comment:
We don't require upgraders for every BC-breaking change that would trigger this test today. For example, ops in prototype and beta modules have no BC guarantees.